<launch>

    <!-- Choose if inertial or non-inertial model is desired -->
    <!-- <node name="vtec_u4_simulation_node" pkg="vanttec_controllers" type="vtec_u4_simulation_node"/> -->
    <node name="vtec_u4_simulation_node" pkg="vanttec_controllers" type="inertial_vtec_u4_simulation_node"/>

    <node name="vtec_u4_asmc_node" pkg="vanttec_controllers" type="vtec_u4_asmc_node">
        <rosparam file="$(find vanttec_controllers)/config/VTec_U4_ASMC_Gains.yaml"/>
    </node>

    <param name="robot_description" textfile="$(find vanttec_controllers)/models/vtec_u3.urdf" />

    <node name="tf2_broadcaster_node" pkg="vanttec_controllers" type="tf2_broadcaster_node">
        <param name="parent_frame" value="world" />
        <param name="child_frame" value="uuv" />
    </node>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find vanttec_controllers)/launch/rviz_cfg/uuv.rviz" />

</launch>